Lower bound on conduit in cereal-conduit.

`cereal-conduit` accepts `conduit >=1.0.0` according to cabal file, but it actually fails to be built

```
Building library for cereal-conduit-0.8.0..
[1 of 2] Compiling Data.Conduit.Cereal.Internal ( Data/Conduit/Cereal/Internal.hs, dist/build/Data/Conduit/Cereal/Internal.o )

Data/Conduit/Cereal/Internal.hs:15:32: error:
    Module ‘Data.Conduit’ does not export ‘ConduitT’
   |
15 | import           Data.Conduit (ConduitT, await, leftover, yield)
   |     
```

Also conduit tells that `ConduitT` data type is available since 1.0.0 https://hackage.haskell.org/package/conduit-1.3.0/docs/Data-Conduit.html but it's actually available since 1.3.0.

Am I missing something, or is there a real issue?

--- a/cereal-conduit/cereal-conduit.cabal
+++ b/cereal-conduit/cereal-conduit.cabal
@@ -17,7 +17,7 @@ extra-source-files: README.md ChangeLog.md
 
 library
     build-depends: base         >= 4.9     && < 5
-                 , conduit      >= 1.0.0   && < 1.4
+                 , conduit      >= 1.3.0   && < 1.4
                  , resourcet    >= 0.4     && < 1.3
                  , cereal       >= 0.4.0.0 && < 0.6
                  , bytestring

--- a/conduit/src/Data/Conduit/Internal/Conduit.hs
+++ b/conduit/src/Data/Conduit/Internal/Conduit.hs
@@ -112,7 +112,7 @@ import Data.Traversable (Traversable (..))
 -- result @r@. The type synonyms provided here are simply wrappers around this
 -- type.
 --
--- Since 1.0.0
+-- Since 1.3.0
 newtype ConduitT i o m r = ConduitT
     { unConduitT :: forall b.
                     (r -> Pipe i i o () m b) -> Pipe i i o () m b

